src/platform/packages/private/kbn-language-documentation/src/components/shared/documentation_content.tsx

```diff
@@ -10,16 +10,50 @@ import React from 'react';
 import { css } from '@emotion/react';
 import { i18n } from '@kbn/i18n';
 import { useEuiTheme, euiScrollBarStyles } from '@elastic/eui';
-import { EuiFlexGroup, EuiText, EuiBetaBadge } from '@elastic/eui';
+import { EuiFlexGroup, EuiFlexItem, EuiText, EuiBetaBadge } from '@elastic/eui';
 import type { LanguageDocumentationSections } from '../../types';
+import { LicenseInfo, MultipleLicenseInfo, getLicensesArray } from '../../utils/get_license_array';
+
+function toTitleCase(str: string): string {
+  return str.charAt(0).toUpperCase() + str.slice(1).toLowerCase();
+}
+function createLicenseTooltip(license: LicenseInfo): string {
+  function startsWithVowel(str: string): boolean {
+    return /^[aeiouAEIOU]/.test(str);
+  }
+
+  let tooltip = i18n.translate('languageDocumentation.licenseRequiredTooltip', {
+    defaultMessage: 'This feature requires {articleType} {license} subscription',
+    values: {
+      license: toTitleCase(license.name),
+      articleType: startsWithVowel(license.name) ? 'an' : 'a',
+    },
+  });
+
+  if (license.isSignatureSpecific && license?.paramsWithLicense?.length) {
+    tooltip += ` ${i18n.translate('languageDocumentation.licenseParamsNote', {
+      defaultMessage: ' only for specific values: {params}',
```

Comment:
```suggestion
      defaultMessage: ' to use the following values: {params}',
```
Suggestion, feel free to ignore if you think it doesn't work for the various possible scenarios